lib/nessy.js

```diff
@@ -14,6 +14,11 @@ module.exports = (selector, value, divider, obj) => {
     
     for (let i = 0; i < arr.length ; i++) {
         const name = arr[i];
+        if (name === '__proto__' ||
+           (name == 'prototype' && i>0 && arr[i-1] == 'constructor')
+          ) {
+            continue;
+        }
```

Comment:
Why don't you use:

```js
if (name === '__proto__')
    continue;

if (name === 'prototype')
    continue;

if (name === 'contructor')
    continue;
```

Also this can be moved out into `notSecure` function. And used with:

```js
if (notSecure)
    continue;